src/NetAppFiles/NetAppFiles/ActiveDirectory/NewNetAppFilesActiveDirectory.cs

```diff
@@ -156,6 +155,18 @@ public class NewAzureRmNetAppFilesActiceDirectory : AzureNetAppFilesCmdletBase
         [ValidateNotNullOrEmpty]
         public SwitchParameter AllowLocalNfsUsersWithLdap { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Domain Users to be added to the Built-in Administrators Active Directory group. A list of unique usernames without domain specifier.")]
+        [ValidateNotNullOrEmpty]
+        public string[] Administrators { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "If enabled, Traffic between the SMB server to Domain Controller (DC) will be encrypted.")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter EncryptDCConnections { get; set; }
```

Comment:
As PowerShell suggest use singularity for parameters. You can find doc [here](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#singularity). Please rename `Administrators` to `Administrator` and `EncryptDCConnections` to `EncryptDCConnection`.

Reply:
updated

Reply:
Do you?

Reply:
hmm, some git issue,  I will have a second look

Reply:
Is updating existing properties allowed ?

Reply:
As your module is not GA yet. Is allowed.